packages/notification-services-controller/src/NotificationServicesController/processors/process-notifications.ts

```diff
@@ -45,6 +50,10 @@ export function processNotification(
     return n;
   }
 
+  if (isSnapNotification(notification)) {
+    return processSnapNotification(notification as RawSnapNotification);
```

Comment:
I'll double check, I think we can ignore the assertion, as we are using assertion functions.